--- xf86-video-r128-6.10.2/src/atipcirename.h.orig	2013-02-07 16:53:52.000000000 +0900
+++ xf86-video-r128-6.10.2/src/atipcirename.h	2018-05-23 21:49:20.300159461 +0900
@@ -35,6 +35,7 @@
 };
 
 #include "xf86Module.h"
+#include <sys/types.h>
 
 #if GET_ABI_MAJOR(ABI_VIDEODRV_VERSION) >= 12
 
@@ -49,10 +50,10 @@
 #endif
 #define PCI_DOMBUS_MASK (((PCI_DOM_MASK) << 8) | 0x0ffu)
 
-static inline uint32_t
+static inline u_int32_t
 pciTag(int busnum, int devnum, int funcnum)
 {
-	uint32_t tag;
+	u_int32_t tag;
 	tag  = (busnum & (PCI_DOMBUS_MASK)) << 16;
 	tag |= (devnum & 0x00001fu) << 11;
 	tag |= (funcnum & 0x000007u) << 8;
