diff -urNp -x '*.orig' xf86-video-r128-6.12.0.org/src/atipcirename.h xf86-video-r128-6.12.0/src/atipcirename.h
--- xf86-video-r128-6.12.0.org/src/atipcirename.h	2018-10-23 06:10:56.000000000 +0200
+++ xf86-video-r128-6.12.0/src/atipcirename.h	2021-11-04 00:00:29.275887352 +0100
@@ -36,6 +36,7 @@ enum region_type {
 
 #include <stdint.h>
 #include "xf86Module.h"
+#include <sys/types.h>
 
 #if GET_ABI_MAJOR(ABI_VIDEODRV_VERSION) >= 12
 
@@ -50,10 +51,10 @@ enum region_type {
 #endif
 #define PCI_DOMBUS_MASK (((PCI_DOM_MASK) << 8) | 0x0ffu)
 
-static inline uint32_t
+static inline u_int32_t
 pciTag(int busnum, int devnum, int funcnum)
 {
-	uint32_t tag;
+	u_int32_t tag;
 	tag  = (busnum & (PCI_DOMBUS_MASK)) << 16;
 	tag |= (devnum & 0x00001fu) << 11;
 	tag |= (funcnum & 0x000007u) << 8;
